.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
     "python.testing.pytestEnabled": true,
     "python.testing.autoTestDiscoverOnSaveEnabled": true,
     "python.testing.promptToConfigure": true,
-    "python.testing.pytestArgs": ["-v", "--color=yes", "--no-cov"],
+    "python.testing.pytestArgs": ["-v", "--color=yes"],
```

Comment:
vscode now allows for a separate button for testing with code coverage.  These arguments now are applied to both coverage and no-coverage.